src/config/chis-tg/config.json

```
@@ -45,11 +129,25 @@
         }
       ],
       "contact_properties": [
+        {
+          "friendly_name": "CHW Last Name",
```

```suggestion
          "friendly_name": "Last Name",
```

Do we need to put CHW here everywhere? Do you think this adds clarity?